cmake/modules/AddSwift.cmake

```diff
@@ -46,12 +46,12 @@ function(_add_variant_c_compile_link_flags
     "-target" "${SWIFT_SDK_${sdk}_ARCH_${arch}_TRIPLE}")
 
   list(APPEND result
-    "-isysroot" "${SWIFT_SDK_${sdk}_PATH}")
+    "-isysroot" "\"${SWIFT_SDK_${sdk}_PATH}\"")
```

Comment:
I'm not deeply attached to any particular strategy, just trying to unblock usage with one of my common configurations.  Alternatives that achieve the same end result are equally welcome.  There may be unnecessary changes here as I had to iterate a bit to get the build process to work at all when my Xcode path contains a space.  I can try to isolate which changes are strictly required and which are just aggressively paranoid.